test/infrastructure/docker/internal/controllers/dockermachine_controller.go

```diff
@@ -291,28 +295,11 @@ func (r *DockerMachineReconciler) reconcileNormal(ctx context.Context, cluster *
 	// Update the BootstrapExecSucceededCondition condition
 	conditions.MarkTrue(dockerMachine, infrav1.BootstrapExecSucceededCondition)
 
-	// set address in machine status
-	machineAddress, err := externalMachine.Address(ctx)
-	if err != nil {
-		log.Error(err, "failed to get the machine address")
+	if err := setMachineAddress(ctx, dockerMachine, externalMachine); err != nil {
+		log.Error(err, "failed to set the machine address")
```

Comment:
didn't change the way it requeues here, since for all other errors in the normal provisioning workflow (pre-move and before the machine is ready) it is requeued after 5 seconds

Reply:
Sounds good. Let's change that in another PR **if** we want to change it